librepo/librepo.h

```diff
@@ -24,10 +24,12 @@
 #include <glib.h>
 
 #include "checksum.h"
+#include "downloader_internal.h"
```

Comment:
I see "downloader_**internal**.h". OK internal API. Why is internal API exposed to public (included to librepo.h)?
Oh. Structures from this internal API are used in python wrapper metadatatarget-py.c. So is it iternal API?